code.js

```diff
@@ -394,10 +395,35 @@ function keyPressed(e) {
             button_number(e.key) 
         }   
     }
+    if (e.key) {
+        key_combination[e.code] = e.key;
+    }
 }
 
 // function to capture keyup events
 function keyReleased(e){
+    if (key_combination['ControlLeft'] && key_combination['KeyV']) {
+        navigator.clipboard.readText().then(text => {
+            box = document.getElementById("box");
+            var isNumber = isFinite(parseInt(text));
+            if (isNumber){
+                var copy_number = parseInt(text)
+                box.innerText = copy_number
+                numbers.push(copy_number)
+            }
```

Comment:
Remove parseInt from line 408, so that users can paste float number too. Also, replace lines 411 and 412 with:

firstNum = true
button_number(copy_number)

Replacing them will allow users to append new numbers after the paste, instead of replacing the pasted number with the user input.